### PR TITLE
[huggingface_pytorch] PT1.13 Training - update for Transformers to 4.26.0

### PR DESCRIPTION
**Pull request by JingyaHuang**
*GitHub Issue #2676:*

### Description

This PR updates Hugginface's PyTorch DLC for training(GPU). Here are the corresponding updated dependencies versions:

* `transformers: 4.26.0`
* `datasets: 2.9.0`
* `torch: 1.13.1`

### Tests run

**NOTE: By default, docker builds are disabled. In order to build your container, please update dlc_developer_config.toml and specify the framework to build in "build_frameworks"**
- [x] I have run builds/tests on commit <INSERT COMMIT ID> for my changes.

**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [x] `sagemaker_local_tests = true`

### DLC image/dockerfile

* `huggingface/pytorch/training/docker/1.13/py3/cu117/Dockerfile.gpu`

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [x] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by tejaschumbalkar — COMMENTED** [review]
Changes LGTM

**Comment by tejaschumbalkar** [comment]
Sanity test will be fixed upon re-release of base PT1.13 image

**Comment by JingyaHuang** [comment]
Hi @tejaschumbalkar, thanks for reviewing! 

What would be the next step for this PR, shall I revert the changes in dlc developer config, and wait for the re-release of the base PT1.13 image?

**Comment by tejaschumbalkar** [comment]
Sanity to be  fixed as a part of #2710

**Comment by dennisbappert** [comment]
Do we have any ETA for the new DLC to be available?

**Comment by tejaschumbalkar** [comment]
Sanity test is green. Please revert the dlc_developer_config.toml file changes to prepare for merge

File: dlc_developer_config.toml [hunk 1]
```diff
@@ -129,4 +129,4 @@ dlc-pr-tensorflow-2-graviton-inference = ""
 # EIA Inference
 dlc-pr-mxnet-eia-inference = ""
 dlc-pr-pytorch-eia-inference = ""
-dlc-pr-tensorflow-2-eia-inference = ""
+dlc-pr-tensorflow-2-eia-inference = ""
```
Comment (tejaschumbalkar) [inline dlc_developer_config.toml:132]: nit: remove spaces and add next line

Comment (JingyaHuang) [reply dlc_developer_config.toml]: It's ok, I found the commit where I submited the changes in the file and reverted it. Thanks for the hint!

**Review by tejaschumbalkar — APPROVED** [review]

**Pull request merged**